src/main/java/se/sundsvall/smloader/integration/util/ErrandConstants.java

```diff
@@ -0,0 +1,19 @@
+package se.sundsvall.smloader.integration.util;
+
+import java.util.Map;
+
+public final class ErrandConstants {
+
+	private ErrandConstants() {
+	}
+
+	//TODO Move to properties or db?
```

Comment:
Act on this TODO.

Reply:
It will be more changes in service before it's ready. I'm still waiting for some requirements. The goal for this pr is to be able testing towards OpenE.

Reply:
Will this stop you from putting these properties in a properties file or a database?

Reply:
No. The idea was to solve it later when requirements are set.